src/napi/napi.zig

```diff
@@ -653,6 +648,7 @@ pub extern fn napi_define_class(
 pub extern fn napi_wrap(env: napi_env, js_object: napi_value, native_object: ?*anyopaque, finalize_cb: napi_finalize, finalize_hint: ?*anyopaque, result: [*]*Ref) napi_status;
 pub extern fn napi_unwrap(env: napi_env, js_object: napi_value, result: [*]*anyopaque) napi_status;
 pub extern fn napi_remove_wrap(env: napi_env, js_object: napi_value, result: [*]*anyopaque) napi_status;
+pub extern fn napi_create_object(env: napi_env, result: *napi_value) napi_status;
```

Comment:
Will need to add this under doNotOptimizeAway at the bottom of this file and to symbols.txt so that macOS sees it

Reply:
is already added